packages/flame/lib/src/camera/camera_component.dart

```diff
@@ -120,6 +122,17 @@ class CameraComponent extends Component {
 
   Viewfinder _viewfinder;
 
+  /// The [considerViewport] flag is read-only and cannot be set except through
+  /// [setBounds] as an optional parameter. If true, this represents that the
+  /// camera's [viewfinder] will recalculate your desired bounds with respect
+  /// to your viewport dimensions by ensuring that [ViewportAwareBoundsBehavior]
+  /// component is mounted correctly when the tree updates.
+  ///
+  /// The default value is false.
+  bool get considerViewport => _considerViewport;
```

Comment:
Maybe this should be clarified in the name that it is related to the viewfinder, since this is in the `CameraComponent`. Or maybe the variable should live in the viewfinder instead?

Reply:
Since it relates to a pre-existing convenience for the end-user (b/c of the effect of `setBounds(considerViewport: true)`), adding a comment that this getter is related to the camera's view finder seems reasonable.

Reply:
This getter didn't exist before though, in the `setBounds` method it is clear that `considerViewport` is related to the bounds, meanwhile it isn't in the general `CameraComponent`.

Reply:
What do you want to name it? The only thing that comes to mind is some variant of `hasBoundsAwareViewport`.

Reply:
Yeah, I can only come up with long names too, so I think it would be more natural to move it to the viewfinder, it's not something that the user should have to access often anyways (if ever).

Reply:
Done

Reply:
Super, thanks!